test/integration/scheduler_perf/scheduler_perf_types.go

```diff
@@ -0,0 +1,30 @@
+package benchmark
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+		"k8s.io/kubernetes/pkg/api/v1"
+)
+
+// High Level Configuration for all predicates and priorities.
+type schedulerPerfConfig struct {
```

Comment:
Composing multiple schedulerPerfConfigs doesn't sound too natural to me. It's probably better to have a slice here if you consider having multiple affinities in future (but this is actually hard problem, I don't really know how to solve correctly).

Generally it'd be great if you could discuss this config with @jeremyeder and @sjug.